### PR TITLE
KAA-1070: Moves memory check to cmake

### PR DESCRIPTION
**Pull request by ohamov**

### CODE REVIEW

**Comment by rasendubi** [comment]
@kaa-leeroy retest this please

File: client/client-multi/client-c/build.sh [hunk 1]
```diff
@@ -52,88 +52,28 @@ build() {
...
-    fi
-
-    if [ $CPPCHECK_INSTALLED -eq 1 ]; then
-        run_cppcheck
-    fi
-
-    if [ $RATS_INSTALLED -eq 1 ]; then
+    cd build-posix
+    ctest -T memcheck
+    ctest -T  coverage
```
Comment (rasendubi) [inline client/client-multi/client-c/build.sh]: spacing

**Comment by ohamov** [comment]
pull-request was re-opened https://github.com/kaaproject/kaa/pull/690

**Pull request closed without merging**